composer.json

```diff
@@ -47,9 +47,7 @@
         "twig/twig": "^2.10 || ^3.0"
     },
     "conflict": {
-        "simplethings/entity-audit-bundle": ">=2.0",
-        "sonata-project/admin-bundle": "<=4.0.0-alpha-2",
-        "sonata-project/block-bundle": "<4.2"
```

Comment:
maybe we should keep this one. If some day admin-bundle does not require block-bundle we might have trouble here. block-bundle is a require-dev dependency here.

Reply:
done